04-onion-routing.md

```diff
@@ -827,9 +833,10 @@ handling by the processing node.
    * [`u64`:`htlc_msat`]
    * [`u32`:`height`]
 
-The `payment_hash` is unknown to the final node, the amount for that
-`payment_hash` is incorrect or the CLTV expiry of the htlc is too close to the
-current block height for safe handling.
+The `payment_hash` is unknown to the final node, the `payment_secret` doesn't
+match the `payment_hash`, the amount for that `payment_hash` is incorrect or
```

Comment:
this is kind of confusing: "the `payment_secret` doesn't match the `payment_hash`

I think what you really mean to convey is that the `payment_secret` corresponds with the expected secret for that `payment_hash`. 

'match' implies equivalence, which isn't the exact case here.

Reply:
True, the wording isn't great. This PR is now included in the MPP once so I'll be closing it anyway ;)